docs/licensing.md

```diff
@@ -1,9 +1,357 @@
+      ![Activation Dialog](images/adn_activation_dialog.png)
+      <figcaption><b>Figure 9:</b> Activation Dialog.</figcaption>
+    </figure>
+
+3. AdonisFX will inform about the amount of days left for the trial license.
+    <figure style="width:80%" markdown>
+      ![Trial Dialog](images/adn_your_trial_expires_dialog.png)
+      <figcaption><b>Figure 10:</b> Trial Dialog.</figcaption>
+    </figure>
+
+4. The trial will deactivate the product after 30 days and will ask for a valid product key.
+5. AdonisFX is activated and ready to be used in trial mode.
+
+**Offline Trial Activation**
+
+This activation mode does *not require access to the internet* for activating the trial mode.
+
+It will allow the user to use AdonisFX for 30 days without providing a `PRODUCT KEY`. Once that trial period is over, the user will be asked to introduce a valid product key.
+
+To activate AdonisFX in Offline Node-Locked Interactive Trial mode:
+
+1. Open `AdonisFX/bin`.
+2. Run `TrialOfflineRequest` from `AdonisFX/bin` with admin privileges.
+3. Enter a valid product key.
+    - The `PRODUCT KEY` has the following format: **XXXX-XXXX-XXXX-XXXX-XXXX-XXXX-XXXX**
+    - A XML file `TrialOfflineRequest.xml` will be generated in the same folder.
+4. Send the request file to **adnsupport@inbibo.co.uk** providing enough information to backtrack the source of the activation request (username and date).
+5. In a maximum of 24h an `TrialOfflineResponse.xml` will be returned to the source e-mail address.
+6. Download and save the response in the same folder `AdonisFX/bin`.
+7. Execute `TrialOfflineResponse` from `AdonisFX/bin` with admin privileges.
+8. The response will be registered and AdonisFX will be ready to be used.
+    - Once the activation period has concluded, a new activation request and response have to be generated.
 
 ## Floating Licensing
 
-Abc
\ No newline at end of file
+This section will explain how to configure, run and set-up the licensing server and AdonisFX for leasing floating licenses when the goal is to not restrict the use of AdonisFX to one single machine. AdonisFX floating licensing system requires the use of a license server in charge of providing, dropping and handling licenses from an active lease "pool". When the amount of requested licenses surpass the amount of licenses purchased for that floating license server no further activations of AdonisFX can be made until a lease is dropped and returned to the lease "pool". If for example 20 licenses had been purchased, then one of them will remain leased until the plug-in is unloaded from the target DCC.
+
+It is possible to run the license server on one operating system and run instances of AdonisFX on a different operating system.
+
+The licensing server is provided and shipped with the installation of AdonisFX for x64 architectures and can be run on Windows, Linux and MacOS. For more builds please visit [https://wyday.com/download/](https://wyday.com/download/).
```

Comment:
This is currently not true. We only shup the .exe for Windows and not for Linux. Do we want to add the other.exe files or just refer to Windows x64 here?

Reply:
We are going to include all binaries for windows and linux (today hopefully). I would remove "MacOS" from the sentence. And yes, we will provide the binaries for 64-bit. Other architectures will have to be downloaded from that link

Reply:
Done!